### PR TITLE
JetpackLogo - now the lightning will be white by default

### LINKED ISSUE #23420: Jetpack connection: Jetpack logo triangles should be white
When you're connecting a Jetpack site, the Jetpack logo shown doesn't have white triangles since it's transparent:

captura de pantalla 2018-03-19 a la s 15 20 43

Issue can be seen at https://wordpress.com/jetpack/connect

@joanrho researched into this and found the issue in this file
https://github.com/Automattic/wp-calypso/blob/master/client/components/jetpack-logo/index.jsx

### PR DESCRIPTION
**Pull request by MichaelArestad**
Fixes #23420 

* I rebuilt the icon to be a little simpler to understand. Now it is composed of basic shapes.
* I added classes for easier targeting
* I updated the three places that were targeting it via class to use the new classNames.

#### Testing
Test on:
* The login page
* https://wordpress.com/jetpack/connect
* JPO CC @joanrho 

#### Before
image

#### After
image

### CODE REVIEW

File: client/components/jetpack-logo/index.jsx [hunk 1]
```diff
@@ -10,10 +10,14 @@ import PropTypes from 'prop-types';
...
+	<g>
+		<path
+			className="jetpack-logo__icon-circle"
+			d="M16,0C7.2,0,0,7.2,0,16s7.2,16,16,16s16-7.2,16-16S24.8,0,16,0z"
+		/>
+		<polygon className="jetpack-logo__icon-triangle" fill="#fff" points="15,19 7,19 15,3 " />
+		<polygon className="jetpack-logo__icon-triangle" fill="#fff" points="17,29 17,13 25,13 " />
+	</g>
 );
 const svgLogo24 = (
```
Comment (sirreal) [inline client/components/jetpack-logo/index.jsx]: Intentional no white?

**Comment by sirreal** [comment]
Oh, and thanks for this. It had been bothering me for a while 🙂

File: client/components/jetpack-logo/index.jsx [hunk 2]
```diff
@@ -10,10 +10,14 @@ import PropTypes from 'prop-types';
...
-		className="jetpack-logo__icon"
-		d="M16,0C7.2,0,0,7.2,0,16s7.2,16,16,16c8.8,0,16-7.2,16-16S24.8,0,16,0z M15.2,18.7h-8l8-15.5V18.7z M16.8,28.8 V13.3h8L16.8,28.8z"
-	/> // eslint-disable-line max-len
+	<g>
+		<path
+			className="jetpack-logo__icon-circle"
+			d="M16,0C7.2,0,0,7.2,0,16s7.2,16,16,16s16-7.2,16-16S24.8,0,16,0z"
+		/>
+		<polygon className="jetpack-logo__icon-triangle" fill="#fff" points="15,19 7,19 15,3 " />
+		<polygon className="jetpack-logo__icon-triangle" fill="#fff" points="17,29 17,13 25,13 " />
```
Comment (sirreal) [inline client/components/jetpack-logo/index.jsx]: I'm an SVG nerd, sorry 🤓 

We might consider a green circle with white polygons above, or the full green shape (with cut-out) with a white circle below. The `<polygon>`s could be fused into a single path above a simple `circle`, might be the best combination of minimization and clarity.

Also the `<g>` wrapper can be `<React.Fragment>`, we just need a way of working with a single element, we don't actually need an SVG `<g>`. Save an HTML element 🙂 

These aren't blockers, just some notes. Feel free to reach out if you're interested in exploring these options.

Comment (MichaelArestad) [reply client/components/jetpack-logo/index.jsx]: @sirreal I thought about fusing them in as they were before, but I think this makes them easier to target and understand for styling purposes.

File: client/components/jetpack-logo/index.jsx (hunk 1 above)
Comment (MichaelArestad) [reply client/components/jetpack-logo/index.jsx]: Yeah. The 24px version is more like a gridicon where it should follow gridicon styles.

**Comment by rickybanister** [comment]
What's the final word on this, ready to merge as-is?

**Comment by MichaelArestad** [comment]
@rickybanister No. I'm changing the default color to green. Should be done shortly.

**Pull request merged**